docs/reference/mapping/types/core-types.asciidoc

```diff
@@ -379,7 +379,9 @@ defaults to `true` or to the parent `object` type setting.
 |`ignore_malformed` |Ignored a malformed number. Defaults to `false`.
 
 |`numeric_resolution` |The unit to use when passed in a numeric values. Possible
```

Comment:
But this isn't what we've done for removal of every other setting in mappings. I think it is more confusing if someone looks up the docs for a quick reference and sees a setting (without noticing the note about 2.0 for this particular setting) and tries using it, and gets an error.